src/backend/gl/gl_common.c

```diff
@@ -181,7 +181,6 @@ void gl_compose(backend_t *base, void *image_data, int dst_x, int dst_y,
 
 	// It's required by legacy versions of OpenGL to enable texture target
 	// before specifying environment. Thanks to madsy for telling me.
```

Comment:
Total passerby comment (I'm not even how I ended up reading this diff lol), but shouldn't this comment be deleted since what it seems to be referring to is also deleted `glEnable(GL_TEXTURE)`?